CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Confluent's Python client for Apache Kafka
 
+## v1.6.1
+
+v1.6.1 is a feature release:
+
+ - KIP-429 - Incremental consumer rebalancing support.
+ - OAUTHBEARER support.
+
+### Fixes
```

Comment:
big thanks! I should have done this, but you have to admit, it was far easier to just _merge_ and let you deal with it :-).